RevenueCatUI/Modifiers/ViewExtensions.swift

```diff
@@ -35,7 +35,7 @@ extension View {
     /// Wraps the 2 `onChange(of:)` implementations in iOS 17+ and below depending on what's available
     @inlinable
     @ViewBuilder
-    public func onChangeOf<V>(
+    internal func onChangeOf<V>(
```

Comment:
I think you can also skip the `internal` here since it's the default, but I won't go against keeping it explicit